.claude/helpers/merge-claude-settings.sh

```
@@ -0,0 +1,98 @@
+# If a file fails to parse, fall back to empty object
+parsed_json=$(npx -y -p json5 node -e "
+const fs = require('fs');
+const JSON5 = require('json5');
+process.argv.slice(1).forEach(file => {
+  try {
+    console.log(JSON.stringify(JSON5.parse(fs.readFileSync(file, 'utf8'))));
+  } catch {
+    console.log('{}');
+  }
+});
+" $settings_files)
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)